recipes/ega-cryptor/meta.yaml

```
@@ -0,0 +1,36 @@
+test:
+  commands:
+    - ega-cryptor -h
+
+about:
+  home: https://ega-archive.org/submission/data/file-preparation/egacryptor/
+  summary: EGACryptor v.2.0.0
+  license: unknown
+  description: |
+    The EGACryptor v.2.0.0 is a JAVA-based application which enables submitters 
+    to produce EGA compliant encrypted files along with files for the encrypted
+    and unencrypted md5sum for each file to be submitted. The application will 
```

yamllint

[error] trailing spaces

(trailing-spaces)